arraycontext/impl/pyopencl/fake_numpy.py

```diff
@@ -175,6 +186,21 @@ def _flatten_array(ary):
                 f"with strides {ary.strides} of {ary.dtype}")
 
 
+def _cl_scalar_list_norm(ary, ord):
+    if ord is None:
+        ord = 2
+
+    from numbers import Number
+    if ord == np.inf:
+        return max(ary)
+    elif ord == -np.inf:
+        return min(ary)
+    elif isinstance(ord, Number) and ord > 0:
+        return sum(iary**ord for iary in ary)**(1/ord)
+    else:
+        raise NotImplementedError(f"unsupported value of 'ord': {ord}")
+
+
```

Comment:
#14 contains a refactor that moves the norm into the base class. Maybe nab that, so that we only have to touch this once?

Reply:
Moved in 2bc76bb with the changes from #14 (hopefully got everything).